actions.yaml

```diff
@@ -0,0 +1,13 @@
+definitions:
+- id: web.test
+  definition:
+    action:
+      target: callback
+      title: test1
+      description: desc1
+      arguments:
+        - name: arg1
+          title: arg1
+      options:
+        - name: opt1
+          title: opt1
```

Comment:
This currencly renders as
```
➜ ./plasmactl                                                                                  
Usage:
  plasmactl [flags]
  plasmactl [command]

Actions:
  web.test           test1: desc1                  <------- Command from actions.yaml

Additional Commands:
  web                Starts web server             <------- Command from plugin.go
```
Meaning command definition lives in 2 places and can differ from each other, which is actually the case in this example:
```
➜ ./plasmactl web --help           
Starts web server

Usage:
  plasmactl web [flags]
```
```
➜ ./plasmactl web.test --help
test1: desc1

Usage:
  plasmactl web.test arg1 [flags]
```
It would be better for command to be definined in a single palce: whether in `actions.yaml` or in `plugin.go`:
- if command is defined in `plugin.go`, then we only need a trick to make it appear as action (and not listed twice)
- if command is defined in `actions.yaml`, then it should not be in `plugin.go` anymore
- In both cases, we need the ability for a command not to be listed. We will handle that later as a follow up

Reply:
Well maybe task name _Run native commands as action_ is misleading

Main issue we need to solve is that plugin commands are missing from webui.
which only lists actions. Most likely there are other ways to solve that than turning plugin commands into actions